src/components/common/TransactionalFolioNumber.vue

```diff
@@ -57,7 +57,7 @@ export default class TransactionalFolioNumber extends Mixins(CommonMixin) {
 
   /** True if this component is valid. */
   get isValid (): boolean {
-    return (!this.doValidate || !!this.$refs.folioNumberInput?.valid)
+    return (!this.doValidate || !this.$refs.folioNumberInput || this.$refs.folioNumberInput.valid)
```

Comment:
The ref is undefined when the input component is not rendered.

Since this is a getter, it may fire before the component is rendered.

In this case, the result should be True.